### DISCUSSION TITLE
With PHP version 8.0-8.x the task board is not displayed correctly

### DISCUSSION THREAD

**Discussion by carlosnewmusic** [kanboard/kanboard#5218] [Installation & Configuration]
### Checklist

- [x] I verified that Kanboard is correctly installed
- [x] I verified that the problem does not come from a plugin
- [x] I verified that the problem is not already reported
- [x] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

### Actual behaviour
with PHP version 8.0.23 and 8.1.11 the task board is not displayed correctly, however with version 7.4 it is.
/?controller=BoardViewController&action=show&project_id=3

PHP 8.0.23
![imagen](https://user-images.githubusercontent.com/19379457/201097234-a4726b89-e483-448d-8245-11e36f5b4768.png)
PHP 8.1.11
![imagen](https://user-images.githubusercontent.com/19379457/201097234-a4726b89-e483-448d-8245-11e36f5b4768.png)

### Expected behaviour
PHP 7.4.30
![imagen](https://user-images.githubusercontent.com/19379457/201096883-b3918dfc-8eee-4d30-b63c-768169b4dae4.png)

to be displayed correctly

### Steps to reproduce
see the board page

### Configuration

- Kanboard version:  1.2.24
- Database type and version:  mysql  5.7.39
- PHP version: 8.0.23-8.x
- OS:  Linux 
- Browser: Firefox 107.0b9

**Comment by fguillot** [comment]
I think the problem is related to some incorrect configuration on your server. The official Kanboard Docker image is currently using PHP 8.1 and that works fine. You should try to look at the log files to see what is the real problem.

**Reply by carlosnewmusic** [reply]
in 8.2 when opening a project in the dashboard view it appears like this in Firefox 112.0b7
![imagen](https://user-images.githubusercontent.com/19379457/228303640-d22e0291-7d6a-4f29-93a4-5372373901ce.png)

and the console displays this message: This page is in Quirks mode. The page layout may be affected. For standard mode, use "<! DOCTYPE html>". https://developer.mozilla.org/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

and the console displays this message in list mode: Using the drawWindow method from CanvasRenderingContext2D is deprecated. Instead use the API for tabs.captureTab extensions https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/captureTab

and the console displays this message in calendar mode: Using the drawWindow method from CanvasRenderingContext2D is deprecated. Instead use the API for tabs.captureTab extensions https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/captureTab

**Reply by fguillot** [reply]
The console logs in your web browser are not going to help you that much. Check the PHP logs and the web server logs (Nginx or Apache depending on what you are using on your server).

**Comment by creecros** [comment]
Highly plausible you have a plugin that does not work with php 8. i would make sure all your plugins are up to date, and that they are all php 8 compatible.
